Objects/descrobject.c

```diff
@@ -1547,6 +1551,7 @@ static PyMemberDef property_members[] = {
     {"fset", _Py_T_OBJECT, offsetof(propertyobject, prop_set), Py_READONLY},
     {"fdel", _Py_T_OBJECT, offsetof(propertyobject, prop_del), Py_READONLY},
     {"__doc__",  _Py_T_OBJECT, offsetof(propertyobject, prop_doc), 0},
+    {"__name__", _Py_T_OBJECT, offsetof(propertyobject, prop_name), 0},
```

Comment:
Should not it be read-only?

Reply:
`__name__` on other things (functions, classes) are writable, so I think this is more consistent. This can be used to assign the name manually when property is added to a class after class construction, since `__set_name__` is not called in this case.